src/Config.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * The class encapsulates properties and behaviour of mosaicConfig and facilitatorConfig.
+ */
+export default class Config {
+  /**
+   * Config class constructor.
+   */
+  public constructor() {
```

Comment:
Do we need a public constructor in the config?   There should a factory or something which should instantiate config possibly from file.

Reply:
this overlaps also with #63 ?

Reply:
PR #63 is merged. Rebased my branch with develop.